added  commands (#1658)
MicrosoftDocs/windows-powershell-docs: docset/windows/dism/Mount-WindowsImage.md

This command remounts a Windows image that has already been mounted to the c:\offline directory but has become inaccessible for servicing.

> ### Example 5: Mount an image at an index of a file with logpath with checkintegrity with optimize

Reviewer: ```suggestion
### Example 5: Mount an image at an index of a file specifying logpath with checkintegrity and optimize parameters
```

```PowerShell
PS C:\> Mount-WindowsImage -Checkintegrity -ImagePath "c:\imagestore\install.wim" -Index 2 -Path "c:\offline" -Logpath C:\install.log -Optimize
```

> This command mounts the Windows image at index 2 of the install.wim file to the c:\offline directory with Checkintegrity and getting a log file with a quicker initial mount time. Additional operations may be slower.

Reviewer: ```suggestion
This command mounts the Windows image at index 2 of the install.wim file to the c:\offline directory with Checkintegrity parameter, requesting a log file to be written and with a quicker initial mount time. Additional operations may be slower.
```

This command remounts a Windows image that has already been mounted to the c:\offline directory but has become inaccessible for servicing.

### Example 5: Mount an image at an index of a file with logpath with checkintegrity with optimize

> ```

Reviewer 2: ```suggestion
```PowerShell
```

PS C:\> Mount-WindowsImage -Checkintegrity -ImagePath "c:\imagestore\install.wim" -Index 2 -Path "c:\offline" -Logpath C:\install.log -Optimize
```

> This command mounts the Windows image at index 2 of the install.wim file to the c:\offline directory with checkintegrity and getting log file with a quicker initial mount time.Additional operations may be slower.

Reviewer 2: ```suggestion
This command mounts the Windows image at index 2 of the install.wim file to the c:\offline directory with Checkintegrity and getting a log file with a quicker initial mount time. Additional operations may be slower.
```

> 

Reviewer 2: ```suggestion
```
